app/assets/javascripts/components/description/DescriptionContent.js

```diff
@@ -24,12 +25,15 @@ const replaceStoryLink = (domNode, linkedStories) => {
 }
 
 const DescriptionContent = ({ description, isReadonly, linkedStories, onClick }) => {
+  const converter = new showdown.Converter();
+  const descriptionHTML = converter.makeHtml(description);
+
   const isEmpty = (!description || !description.length);
```

Comment:
You really need this `( )`, it's a project's pattern?

Reply:
It was like this but I removed now.